### DISCUSSION TITLE
bin/puma-wild: Is it still useful?

### DISCUSSION THREAD

**Discussion by olleolleolle** [puma/puma#3433] [General]
I got curious about the bin/puma-wild file. It's not mentioned in documentation, and scant mention in the code, also.

Can you think of a current-era use case for it?

**Comment by nateberkopec** [comment]
I think when this was written, Bundler was probably less of a thing than it is now. I don't know really anyone who would need Puma to manage their LOAD_PATHs...

Could think of removing it.

**Comment by dentarg** [comment]
Looks like it is used by [the `prune_bundler` feature](https://github.com/puma/puma/blob/62f7cd6fbd9bffa3564cba0d111abe2d27f2b2dc/lib/puma/dsl.rb#L960-L980).

It is referenced in https://github.com/puma/puma/blob/62f7cd6fbd9bffa3564cba0d111abe2d27f2b2dc/lib/puma/launcher/bundle_pruner.rb#L16-L44 

and the tests for it https://github.com/puma/puma/blob/62f7cd6fbd9bffa3564cba0d111abe2d27f2b2dc/test/test_bundle_pruner.rb#L48
